src/components/buttons/base-button-wrapper/base-button-wrapper.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Spacings from '../../../materials/spacings';
+import styles from './base-button-wrapper.mod.css';
+
+// This holds the logic of the `button` element, which wraps a `span` (defined as `BaseButtonContent`) with styles
+class BaseButtonWrapper extends React.Component {
+  static displayName = 'BaseButtonWrapper';
+
+  static propTypes = {
+    id: PropTypes.string,
+    name: PropTypes.string,
+    ref: PropTypes.string,
+    children: PropTypes.node,
+    ariaLabel: PropTypes.string.isRequired,
+    onClick: PropTypes.func.isRequired,
+    dataAttr: PropTypes.object,
+    isDisabled: PropTypes.bool,
+    isToggled: PropTypes.bool,
+    type: PropTypes.oneOf(['button', 'reset', 'submit']),
+  };
+
+  render() {
+    return (
+      <button
+        id={this.props.id}
+        name={this.props.name}
+        disabled={this.props.isDisabled}
+        onClick={this.props.onClick}
```

Comment:
What about all the other `on` events? I have a feeling it might be best to just spread all props and then overwrite the specific ones (or set defaults depending on the attribute).